R-package/src/install.libs.R

```diff
@@ -1,7 +1,14 @@
 # User options
 use_precompile <- FALSE
 use_gpu <- FALSE
+
+# Package will be built with Visual Studio unless you set one of these to TRUE
```

Comment:
Change to the following (or similar, at least specifying Windows requirement):

```
# Under Windows, the package will be built using Visual Studio unless you set one of these to TRUE
```

Reply:
fixed in https://github.com/microsoft/LightGBM/pull/3065/commits/d1d8022cc36b5bff616192474a938861fcb7afce